src/Cake.Web/App_Data/docs.xml

```diff
@@ -35,7 +35,7 @@
     <item title="Visual Studio Code" file="editors/vscode.md" />
   </section>
   <section id="build-systems" title="Build Systems">
-    <item title="Visual Studio Online" file="build-systems/vso.md" hidden="true"/>
```

Comment:
There is support in the site for legacy redirects. Maybe you can add it there and just redirect it to the new URI?